pkg/verification/service/verification_service.go

```diff
@@ -149,15 +153,10 @@ func (s *ServiceImpl) InitVerification(ctx *gin.Context, userID, username, e164P
 
 		// Generate the verification message with the new verification code
 		content := fmt.Sprintf(cfg.Verification().MessageTemplate(), verificationCode)
-		client := twilio.NewClient(cfg.Verification().TwilioAccountSID(), cfg.Verification().TwilioAuthToken(), s.HTTPClient)
-		fromNumber := cfg.Verification().TwilioFromNumber()
-		msg, err := client.Messages.SendMessage(fromNumber, e164PhoneNumber, content, nil)
+
+		err = s.NotificationService.SendNotification(ctx, content, e164PhoneNumber)
 		if err != nil {
-			if msg != nil {
-				log.Error(ctx, err, fmt.Sprintf("error while sending, code: %d message: %s", msg.ErrorCode, msg.ErrorMessage))
-			} else {
-				log.Error(ctx, err, "unknown error while sending")
-			}
+			log.Error(ctx, err, "unknown error while sending")
 
 			// If we get an error here then just die, don't bother updating the UserSignup
 			return crterrors.NewInternalError(err, "error while sending verification code")
```

Comment:
If we get past this point should we have an INFO level message to indicate sending the verification code was sent successfully?

Reply:
I don't think we need it, we seemingly didn't have any logging like this before.